### PR TITLE
Connect regional partner and thanks

### PR DESCRIPTION
**Pull request by mehalshah**
https://trello.com/c/WPUHbHAq/131-regional-partner-contact-form-for-administrators

### CODE REVIEW

File: dashboard/app/controllers/pd/regional_partner_contact_controller.rb [hunk 1]
```diff
@@ -11,4 +8,11 @@ def new
       }.to_json
     }
   end
+
+  # Get /pd/regional_partner_contacts/:contact_id/thanks
+  def thanks
+    regional_partner_contact = Pd::RegionalPartnerContact.find(params[:contact_id])
```
Comment (aoby) [inline dashboard/app/controllers/pd/regional_partner_contact_controller.rb]: can (we use) cancan(can) to load this via `load_resource`? :p

File: dashboard/app/models/pd/regional_partner_contact.rb [hunk 2]
```diff
@@ -70,4 +72,28 @@ def validate_district_fields
...
+    possible_partners = school_district.regional_partners_school_districts
+
+    if possible_partners.size == 1
+      self.regional_partner = possible_partners.first.regional_partner
+    elsif possible_partners.size > 1
+      grade_levels = sanitize_form_data_hash[:grade_levels]
+
+      if grade_levels.include? 'High School'
+        self.regional_partner = possible_partners.find_by(course: 'csp').regional_partner
+      elsif grade_levels.include? 'Middle School'
```
Comment (aoby) [inline dashboard/app/models/pd/regional_partner_contact.rb:92]: should these be in constants? Where are the values provided for the form?

File: dashboard/app/controllers/pd/regional_partner_contact_controller.rb (hunk 1 above)
Comment (mehalshah) [reply dashboard/app/controllers/pd/regional_partner_contact_controller.rb]: Yes we can(can(can))

File: dashboard/app/models/pd/regional_partner_contact.rb (hunk 2 above)
Comment (mehalshah) [reply dashboard/app/models/pd/regional_partner_contact.rb:92]: They come from regional_partneres_school_districts. I can make them a constant

**Pull request merged**